zanata-client-commands/src/main/java/org/zanata/client/commands/init/SourceConfigPrompt.java

```diff
@@ -117,6 +125,24 @@ SourceConfigPrompt promptUser() throws Exception {
         pushOptions.setSrcDir(srcDir);
         pushOptions.setIncludes(includes);
         pushOptions.setExcludes(excludes);
+
+
+        // if project type is file, we need to ask file type in order to find source documents
+        if (File.name().equalsIgnoreCase(pushOptions.getProjectType())) {
+
+            console.blankLine();
+            console.printfln(Question, get("project.file.type.question"));
+            RestClientFactory clientFactory = getClientFactory(pushOptions);
+            FileResourceClient client =
+                    clientFactory.getFileResourceClient();
+            rawDocumentTypes = client.acceptedFileTypes();
+            // this answer is not persisted in zanata.xml so user will still need to type it when they do the actual push
+            console.printfln(Hint, PushOptionsImpl.fileTypeHelp);
```

Comment:
No... initially I was planning to show a list and let user choose. But then I realized it's a multiple choice question (plus optional extension) and it's too hard to implement. So now I just reuse the help message not the actual result fetched from server. I will remove this.